js/Store.js

```diff
@@ -45,7 +61,19 @@ function beginTurn(state, player) {
   };
 }
 
-function damage(state, cardId, amount) {
+function damagePlayer(state, player, amount) {
+  state = {
+    ...state,
+    life: state.life.update(player, m => m - amount),
+  };
+  return {
```

Comment:
This extra line seems like it doesn't do anything. Can you just return from the previous line?